data/intro missions.txt

```diff
@@ -451,3 +451,143 @@ conversation "end of intro missions"
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro: Follow Up - Hai Before Main Plot"
+	name "Bring James to <planet>"
+	description "Take James to <destination> to meet the Hai."
+	blocked `You consider paying a visit to James to tell him about the Hai, but decide to wait until you have a free seat available on your ship. Just in case.`
+	passengers 1
+	landing
+	source "Hestia"
+	destination "Allhome"
+	to offer
+		or
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Interceptor]: done"
+		has "First Contact: Hai: offered"
+	to fail
+		has "chosen sides"	
+	on offer
+		conversation
+			`As you're coming in for a landing, you remember that James left you an open invitation to look him up. Do you want to check in and see how he is doing?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	When you contact James to tell him that you are on visiting <origin>, he is happy to come out to the starport and meet you. The two of you talk a bit about the business of running ships and before long the conversation comes around to the Hai. James is not completely surprised as your story confirms rumors that he has already heard.`
+			choice
+				`	"Why don't you come with me and see them for yourself?  There are a lot of retired spacers on <planet>."`
+				`	"It has been good catching up with you, James."`
+					goto end
+			`	James agrees. When you ask if he needs to collect any of his belongings before the trip he declines, but buys a simple traveling hygiene kit from one of the spaceport shops.`
+				accept
+			label end
+			`	You finish your conversation and go on your way`
+				decline
+	on decline
+		log `Paid a visit to James on <origin> and told him about the Hai.`
+	on complete
+		conversation
+			`James is impressed by the friendly aliens and humans living side by side on <planet>. While you are introducing him to one of the Hai, an old spacer recognizes James and offers to make further introductions around the community on <planet>. You make your goodbyes once again, and prepare to continue on your way.`
+		log `Paid a visit to James on <origin> to tell him about the Hai. Brought him to <planet> afterwards.`
+
+
+
+mission "Intro: Follow Up - After Main Plot at Allhome"
+	invisible
+	landing
+	source "Allhome"
+	to offer
+		has "Intro: Follow Up - Hai Before Main Plot: done"
+		has "main plot completed"
+	to fail
+		has "Intro: Follow Up - Hai After Main Plot"
```

Comment:
This needs some sort of condition to it (i.e. `: done`, `: offered` or `:active`). Also, no need for a `to fail` on a mission that is never accepted.

Reply:
corrected to `: offered`